src/main/webapp/src/App.js

```diff
@@ -0,0 +1,210 @@
+import React, { Component } from 'react';
+import {
+  MDBNavbar,
+  MDBNavbarBrand,
+  MDBNavbarNav,
+  MDBNavbarToggler,
+  MDBCollapse,
+  MDBNavItem,
+  MDBFooter,
+  MDBNavLink,
+  MDBTooltip,
+  MDBIcon
+} from 'mdbreact';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { ReactComponent as Logo } from './assets/logo.svg';
+import Routes from './Routes';
+
+class App extends Component {
+  state = {
+    collapseID: ''
+  };
+
+  toggleCollapse = collapseID => () =>
+    this.setState(prevState => ({
+      collapseID: prevState.collapseID !== collapseID ? collapseID : ''
+    }));
+
+  closeCollapse = collID => () => {
+    const { collapseID } = this.state;
+    window.scrollTo(0, 0);
+    collapseID === collID && this.setState({ collapseID: '' });
+  };
+
+  render() {
+    const overlay = (
+      <div
+        id='sidenav-overlay'
+        style={{ backgroundColor: 'transparent' }}
+        onClick={this.toggleCollapse('mainNavbarCollapse')}
+      />
+    );
+
+    const { collapseID } = this.state;
+
+    return (
+      <Router>
+        <div className='flyout'>
+          <MDBNavbar color='indigo' dark expand='md' fixed='top' scrolling>
+            <MDBNavbarBrand href='/' className='py-0 font-weight-bold'>
+              <Logo style={{ height: '2.5rem', width: '2.5rem' }} />
+              <strong className='align-middle'>MDB React</strong>
+            </MDBNavbarBrand>
+            <MDBNavbarToggler
+              onClick={this.toggleCollapse('mainNavbarCollapse')}
+            />
+            <MDBCollapse id='mainNavbarCollapse' isOpen={collapseID} navbar>
+              <MDBNavbarNav right>
+                <MDBNavItem>
+                  <MDBNavLink
+                    exact
+                    to='/'
+                    onClick={this.closeCollapse('mainNavbarCollapse')}
+                  >
+                    <strong>Home</strong>

```

Comment:
Can we only keep the nav items that we need: about and help. and have log in as a button?

Reply:
Yes, I'll add it in the next pull request.